index.html

```diff
@@ -761,15 +761,15 @@ <h3>Window Roles</h3>
 	</section>
 	<section id="role_definitions">
 		<h2>Definition of Roles</h2>
-		<p>Below is an alphabetical list of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a>roles</a> to be used by rich internet application authors.</p>
+		<p>Below is an alphabetical list of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a>roles</a> to be used by authors.</p>
 		<p>Abstract roles are used for the ontology. Authors MUST NOT use abstract roles in content.</p>
 		<p id="index_role">Placeholder for compact list of roles</p>
 		<div class="role" id="alert">
 			<rdef>alert</rdef>
 			<div class="role-description">
 				<p>A type of <a>live region</a> with important, and usually time-sensitive, information. See related <rref>alertdialog</rref> and <rref>status</rref>.</p>
 				<p>Alerts are used to convey messages to alert the user. In the case of audio warnings this is an accessible alternative for a hearing-impaired user. The <code>alert</code> <a>role</a> goes on the node containing the alert message. Alerts are specialized forms of the <rref>status</rref> role, which will be processed as an atomic <a>live region</a>.</p>
```

Comment:
I've never liked the overuse of the word alert here; it is like using a word to define itself. I know this is not directly related to the purpose of this PR, but it is nonetheless editorial clarification, which is in the spirit of the PR.

In addition, there is a missing comma in the second sentence that I fixed in this suggestion.

```suggestion
				<p>Alerts are used to convey messages that may be immediately important to users. In the case of audio warnings, alerts provide an accessible alternative for hearing-impaired users. The <code>alert</code> <a>role</a> is applied to the element containing the alert message. An <code>alert</code> is a specialized form of the <rref>status</rref> role, which is processed as an atomic <a>live region</a>.</p>
```